release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -1571,22 +1571,101 @@ With this release, availability sets are no longer modified after creation, allo
 [id="ocp-release-note-driver-toolkit-bug-fixes_{context}"]
 ==== Driver ToolKit (DTK)
 
+
+
 [discrete]
 [id="ocp-release-note-cloud-etcd-operator-bug-fixes_{context}"]
 ==== etcd Cluster Operator
 
 [discrete]
-[id="ocp-release-note-image-registry-bug-fixes_{context}"]
-==== Registry
+[id="ocp-release-note-image-streams-bug-fixes_{context}"]
+==== ImageStreams
+
+* Previously, image port blocked registries that would fail if those registries were configured with `NeverContactSource`, even when mirror registries were set up. With this update, image importing is no longer blocked when a registry has mirrors configured. This ensures that image imports succeed even if the original source was set to `NeverContactSource` in the `ImageDigestMirrorSet` or `ImageTagMirrorSet` resources. (link:https://issues.redhat.com/browse/OCPBUGS-44432[*OCPBUGS-44432*]
```

Comment:
Missing parentheses, not bold,  and I think it's supposed to be import
```suggestion
* Previously, image import blocked registries that would fail if those registries were configured with `NeverContactSource`, even when mirror registries were set up. With this update, image importing is no longer blocked when a registry has mirrors configured. This ensures that image imports succeed even if the original source was set to `NeverContactSource` in the `ImageDigestMirrorSet` or `ImageTagMirrorSet` resources. (link:https://issues.redhat.com/browse/OCPBUGS-44432[OCPBUGS-44432])
```